x-pack/plugins/canvas/server/lib/query_es_sql.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { map, zipObject } from 'lodash';
+// @ts-expect-error
+import { buildBoolArray } from './build_bool_array';
+// @ts-expect-error
+import { sanitizeName } from './sanitize_name';
+// @ts-expect-error
+import { normalizeType } from './normalize_type';
```

Comment:
These looks like very cheap conversion opportunities... can we include them as TS in this PR?